docs/paas/deployment_tools.md

```diff
@@ -0,0 +1,280 @@
+---
+id: deployment-tools
+title:  Deployment tools
+sidebar_label: Deployment tools
+---
+
+# Deployment tools
+
+Kubernetes offers, by design, two deployment strategies:
+
+* Recreate &rarr; terminates the pod and, once terminated, scales up the new one;
```

Comment:
Why `&rarr;` generally we would simply use a colon `:`

Reply:
up

Reply:
it's not a matter of what I like lol, we just don't do it in any other bullet list in the repo :)

Reply:
I replaced the beautiful `&rarr;` with boring colons.